docs/sources/enterprise/license/activate-aws-marketplace-license/activate-license-on-ecs.md

```diff
@@ -10,70 +10,95 @@ weight = 250
 
-Update the [database]({{< relref "../../../../../administration/configuration.md#database" >}}) configuration section, so that Grafana use a shared database for storing dashboard, users, and other persistent data.
+   For example, enter `grafana/grafana-enterprise:8.3.3`.
 
-> If you do not have a MySQL database already, [create an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) and check the information needed to [connect to your RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_CommonTasks.Connect.html).
+   > Only Grafana Enterprise versions 8.3.0 and later support licenses granted through AWS Marketplace.
 
-Create a new revision of the Task Definition for the ECS Task that runs Grafana Enterprise. For details, refer to the AWS documentation on [updating a task definition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/update-task-definition.html).
+## Task 2: Configure Grafana for high availability
 
-Within the new revision, edit the Grafana Enterprise container for this task, and add the following environment variable to the container:
+Grafana requires that you configure a database to hold dashboards, users, and other persistent data.
 
-```
-GF_DATABASE_TYPE=mysql
-GF_DATABASE_HOST=localhost:3306
-GF_DATABASE_NAME=grafana
-GF_DATABASE_USER=[mysql database username]
-GF_DATABASE_PASSWORD=[mysql database password]
-```
+### Before you begin
 
-> For more information about how to update your ECS service with an environment variable, refer to [Updating a service using the new console](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/update-service-console-v2.html).
+- Ensure that you have a MySQL database available. For information about creating a MySQL database, refer to [Creating an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html).
+- Review the information required to connect to the RDS DB instance. For more information, refer to [Connecting to an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_CommonTasks.Connect.html).
+
+To configure Grafana for high availability:
+
+1. In Grafana Enterprise, open the grafana.ini file and update the [database] parameters.
+
+   For more information about configuring the database, refer to [Configuration]({{< relref "../../../administration/configuration.md#database" >}}).
+
+1. Create a revision of the task definition for the ECS Task that runs Grafana Enterprise.
+
+   For more information about creating a task, refer to [Updating a task definition using the classic console](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/update-task-definition.html).
+
+1. Within the new revision, edit the Grafana Enterprise container for this task, and add the following environment variable to the container:
+
+   ```
+   GF_DATABASE_TYPE=mysql
+   GF_DATABASE_HOST=localhost:3306
```

Comment:
Maybe we should use `[database address and port]`

Reply:
Done, please check.